mlrun/runtimes/daskjob.py

```diff
@@ -683,6 +683,7 @@ def get_obj_status(selector=[], namespace=None):
 
 class DaskRuntimeHandler(BaseRuntimeHandler):
     kind = "dask"
+    class_modes = {"run": "dask"}
```

Comment:
```suggestion
    class_modes = {RuntimeClassMode.run: "dask"}
```
Continuing the comment, why not use the enum?